prac_09/band.py

```diff
@@ -0,0 +1,22 @@
+class Band:
+    """Represent a Band object."""
+    def __init__(self, name=""):
+        """Initialize a Band instance."""
+        self.name = name
+        self.members = []
+
+    def __str__(self):
+        """Return a string representation of a Band object."""
+        member_strings = [str(member) for member in self.members]
+        return f"{self.name} ({', '.join(member_strings)})"
```

Comment:
This works well but could be condensed to something like:
return f"{self.name} ({', '.join([str(member) for member in self.member])})"
I understand if this looks harder to read so which ever way you prefer should be fine.